src/Symfony/Bundle/FrameworkBundle/Command/RouterApacheDumperCommand.php

```diff
@@ -74,7 +74,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        trigger_error('The router:dump-apache command is deprecated since 2.5 and will be removed in 3.0', E_USER_DEPRECATED);
+        trigger_error('The router:dump-apache command is deprecated since version 2.5 and will be removed in 3.0', E_USER_DEPRECATED);
```

Comment:
here, it will probably be better to display a warning in the output instead given that it is a command